features/exploredash/src/main/java/org/dash/wallet/features/exploredash/ui/ctxspend/CTXSpendViewModel.kt

```diff
@@ -126,7 +126,7 @@ class CTXSpendViewModel @Inject constructor(
 
             val response = repository.purchaseGiftCard(
                 merchantId = it,
-                fiatAmount = amountValue.toBigDecimal().toDouble().toString(),
+                fiatAmount = MonetaryFormat.FIAT.noCode().format(amountValue).toString(),
```

Comment:
If we had a fiat value of 0.00999981 that should be 0.01

`amountValue.toBigDecimal().toDouble().toString()` would give 0.00999981.  CTX would truncate that to 0.00

`MonetaryFormat.FIAT.noCode().format(amountValue).toString()` will give 0.01